samples/drivers/spi_flash_at45/src/main.c

```diff
@@ -32,20 +30,26 @@ void main(void)
 	int i;
 	int err;
 	uint8_t data;
+#if IS_ENABLED(CONFIG_FLASH_PAGE_LAYOUT)
```

Comment:
Generally this should be:
```
#ifdef CONFIG_FLASH_PAGE_LAYOUT
```
though I see existing practice in this drver was to use the `IS_ENABLED()` variant.  So it's okay here, but in any driver that doesn't deviate from standard it should be `#ifdef`.

Reply:
replace `IS_ENABLED()` by `#ifdef`